tests/python/upgrade_3rdParty.sh

```diff
@@ -0,0 +1,15 @@
+rm -rf 3rdParty 
```

Comment:
- Please add header, information what the file does, shebang. 
- Does it need a .bat equivalent?
- can we add more printouts to explain what is going on
- rather than doing git commands - could we print instructions to say "now you need to add files to bit by doing `git add ....` - it always freaks me out a bit when something happens to git and it wasn't me